go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/kedacore/keda/v2
 
-go 1.22.1
-
-toolchain go1.23.3
+go 1.23.3
```

Comment:
Please, revert this change. We decided to keep `go` version as the minimum required and defining `toolchain` with the required one -> https://github.com/kedacore/keda/pull/6378